requirements.txt

```diff
@@ -1,11 +1,11 @@
-tqdm==4.65.0
-openai==0.27.8
+tqdm>=4.65.0
+openai>=0.27.8
 tiktoken
-transformers==4.31.0
-pyserini==0.22.0
-python-dotenv==1.0.0
-faiss-gpu==1.7.2
-faiss-cpu==1.7.2
-ftfy==6.1.1
-fschat==0.2.28
-accelerate==0.23.0
\ No newline at end of file
+transformers>=4.31.0
```

Comment:
wouldn't this have to be a higher version minimum to accommodate mistral based models?

Reply:
Fixed it